tests/mock_tests/aclorch_ut.cpp

```diff
@@ -628,16 +628,74 @@ namespace aclorch_test
         // consistency validation with CRM
         bool validateResourceCountWithCrm(const AclOrch *aclOrch, CrmOrch *crmOrch)
         {
-            auto resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
-            auto ifpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_INGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
-            auto efpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_EGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
+             // ACL counter
```

Comment:
Can you be more specific? Maybe "Verify ACL tables" to be consistent with your comment below.

Reply:
yes, will fix it